src/OrderHistory.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { setUsersThunk, setProductsThunk, setOrdersThunk, setOrderProductsThunk } from './store';
+
+
+
+class _OrderHistory extends React.Component {
+  constructor(props) {
+    super();
+  }
+  async componentDidMount() {
+    await this.props.getUsers()
+    await this.props.getProducts()
+    await this.props.getOrders()
+    await this.props.getOrderProdcuts()
```

Comment:
This is fine, but as the app grows, we probably want to get the data as few times as possible, right now a lot of components ask for all this data.